### PR TITLE
Create script to test feature flags

### LINKED ISSUE #73: Test builds with only some features enabled
Some of our packages do not build properly with only some of their features enabled.
We should test and fix this somehow.

### PR DESCRIPTION
**Pull request by paberr**
## Pull request checklist

- [x] All tests pass. Demo project builds and runs.
- [x] I have resolved any merge conflicts.

Fixes a bunch of errors induced by missing feature flags
Closes #73

### CODE REVIEW

File: scripts/test_features.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import glob
```
Comment (riptl) [inline scripts/test_features.py:2]: `#!/usr/bin/env python3` + `chmod 755`? :stuck_out_tongue:

File: scripts/test_features.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+        try:
+            build_with_features(path, [feature], mode=mode)
+        except subprocess.CalledProcessError as e:
+            print_err('Failed to build feature {} for package at {}'.format(feature, path))
+            print_err(e.stderr)
+            sys.exit(2)
+
+
+if __name__ == '__main__':
```
Comment (riptl) [inline scripts/test_features.py:71]: Could you add a GitLab CI job to the test stage that only executes on scheduled pipelines?
```yml
check-features:
  stage: test
  only:
    - schedules
  script: ...
```

**Review by riptl — DISMISSED** [review]

File: scripts/test_features.py (hunk 2 above)
Comment (paberr) [reply scripts/test_features.py:71]: Sure, I'll do it then.

**Review by jgraef — APPROVED** [review]
:+1:

**Pull request merged**